style.css

```diff
@@ -0,0 +1,476 @@
+  gap: 8px;
+  width: 117px;
+  height: 24px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.headline h3 a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.abt_head .btn a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  letter-spacing: 0.03em;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.container h3 a:hover {
+  color: red;
+}
+
+.work > ul {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  height: 2788px;
+  left: 0;
+  top: 718px;
+}
+
+.work > ul > li {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work > ul > li > div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 295px;
+  height: 288px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.work > ul > li > div > div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 295px;
+  height: 80px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work > ul > li > div > div h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.itm1 ul {
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin: 4px;
+  gap: 16px;
+}
+
+.work > ul > li > div > div ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  height: 32px;
+}
+
+.work > ul > li > div > div ul li:nth-of-type(1) {
+  color: #344563;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.work > ul > li > div > div ul li:nth-of-type(3) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  width: 88px;
+  height: 16px;
+}
+
+.work > ul > li > div > div ul li:nth-of-type(5) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.work > ul > li > div > div > ul > li img {
+  width: 8px;
+  height: 8px;
+}
+
+.about > p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work > ul > li > div > p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  overflow: hidden;
+  width: 203px;
+  height: 24px;
+}
```

Comment:
- [ ] **[Optional]** I think it would be better to improve the design on languages and try to match that of Figma design :joy: 

#### Your design
![image](https://user-images.githubusercontent.com/102757126/205422327-5172600d-0ecc-476b-931f-494b1f521163.png)
#### Figma design
![image](https://user-images.githubusercontent.com/102757126/205422344-c0ecc534-0efb-49ae-959c-5ebabafa929a.png)